plugins/module_utils/network/sonic/argspec/interfaces/interfaces.py

```diff
@@ -57,6 +57,7 @@ def __init__(self, **kwargs):
                                       "SPEED_40GB",
                                       "SPEED_50GB",
                                       "SPEED_100GB",
+                                      "SPEED_200GB",                                     
```

Comment:
```suggestion
                                      "SPEED_200GB",
```
Remove trailing whitespaces after this line please